client/src/components/AlumniDirectory.js

```diff
@@ -162,18 +170,27 @@ export default class AlumniDirectory extends Component {
 
     requestVisible(post, i) {
```

Comment:
A couple of logic changes here:
Students get a specific button that is disabled if an alumni lacks a link or topics
Alumni get a button that is always active